exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporter.java

```diff
@@ -117,7 +117,8 @@ private void flushAndReschedule() {
   }
 
   private void scheduleDelayedFlush() {
-    controller.scheduleTask(Duration.ofSeconds(configuration.bulk.delay), this::flushAndReschedule);
+    controller.scheduleCancellableTask(
```

Comment:
Same, what happens if we don't cancel the task on close? Would it still be called? Would it throw errors?

Reply:
The same as before right ? It uses the same logic

Reply:
Fair enough - I'm not sure if it could be an issue, but I guess it's out of scope.